zstacklib/zstacklib/utils/ovn.py

```
@@ -547,6 +660,106 @@ def getTableAttr(self, table, object, attr):
+        """Return a sorted list of interface names belonging to a port (bond).
+
+        Queries the OVSDB Port table for the interface UUIDs, then resolves
+        each UUID to an interface name.  Returns an empty list on error.
+        """
+        err, iface_uuids_raw = self.getTableAttr('port', port_name, 'interfaces')
+        if err or not iface_uuids_raw:
+            return []
+        # iface_uuids_raw is e.g. "[uuid1, uuid2]" or a single uuid
+        uuids = [u.strip() for u in iface_uuids_raw.strip('[]').split(',') if u.strip()]
+        names = []
+        for uuid in uuids:
```

Ruff (0.15.6)

[error] Import `uuid` from line 8 shadowed by loop variable

(F402)